components/style/context.rs

```diff
@@ -364,7 +364,7 @@ impl<E: TElement> SelectorFlagsMap<E> {
     pub fn new() -> Self {
         SelectorFlagsMap {
             map: FnvHashMap::default(),
-            cache: LRUCache::new(4),
+            cache: LRUCache::<[_;4]>::new(),
```

Comment:
Just tested it, you're right, it was completely superfluous.

I will never get used to how smart Rust is...